cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -112,6 +112,31 @@ spec:
         severity: critical
   - name: osd-alert.rules
     rules:
+    - alert: CephOSDCriticallyFull
+      annotations:
+        description: Storage device {{ $labels.ceph_daemon }} utilization has crossed
+          85% on host {{ $labels.hostname }}. Immediately free up some space or contact
+          support.
+        message: Storage device is critically full.
+        severity_level: error
+        storage_type: ceph
+      expr: |
+        (ceph_osd_metadata * on (ceph_daemon) group_left() (ceph_osd_stat_bytes_used / ceph_osd_stat_bytes)) >= 0.85
+      for: 40s
+      labels:
+        severity: critical
+    - alert: CephOSDNearFull
+      annotations:
+        description: Storage device {{ $labels.ceph_daemon }} utilization has crossed
+          75% on host {{ $labels.hostname }}. Free up some space or contact support.
```

Comment:
```suggestion
          75% on host {{ $labels.hostname }}. Free up some space or contact support or increase capacity.
```

Reply:
I am not sure increasing capacity would currently help.
We can increase capacity by adding OSDs.
But the point here is that in a case of inhomogenious pg distribution, single OSDs would be filled much more than others.

Reply:
I take this back. adding capacity should be recommended, and could help a bit if done early enough...

Reply:
Exactly and that is why I did not add it as it does not solve the issue.